### PR TITLE
EGK LW20W00

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/enginko/EGK-LW20W00/distance.schema.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Distance",
+  "type": "object",
+  "properties": {
+    "adc": {
+      "title": "ADC",
+      "type": "integer",
+      "description": "Analog to digital Converter"
+    },
+    "batteryLevel": {
```
Comment (mariusgiger) [inline types/enginko/EGK-LW20W00/distance.schema.json:12]: why is this in distance?

File: types/enginko/EGK-LW20W00/distance.schema.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "$id": "https://akenza.io/enginko/EGK-LW20W00/distance.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Distance",
```
Comment (mariusgiger) [inline types/enginko/EGK-LW20W00/distance.schema.json:4]: fill_level might be more accurate as the topic name, what do you think?

File: types/enginko/EGK-LW20W00/uplink.js [hunk 3]
```diff
@@ -0,0 +1,464 @@
...
+}
+
+function isNumeric(str) {
+  return /^\-?[0-9]+(e[0-9]+)?(\.[0-9]+)?$/.test(str);
+}
+
+function consume(event) {
+  const payload = event.data.payloadHex;
+  const uplinkId = payload.substring(0, 2);
+  let topic = "default";
```
Comment (mariusgiger) [inline types/enginko/EGK-LW20W00/uplink.js:428]: there is no default topic specified

**Review by mariusgiger — COMMENTED** [review]
looks good, one small comment

File: types/enginko/EGK-LW20W00/distance.schema.json (hunk 2 above)
Comment (J2thatsme) [reply types/enginko/EGK-LW20W00/distance.schema.json:4]: Filllevel is only accurate if its configured, so i think distance would be a save bet.

Comment (mariusgiger) [reply types/enginko/EGK-LW20W00/distance.schema.json:4]: Distance is only one of three right? so it might not be the best name

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**